packages/components/pie-switch/test/visual/pie-switch.spec.ts

```diff
@@ -38,3 +38,22 @@ test.describe('Prop: `Label`', () => {
         });
     });
 });
+
+test.describe('when in RTL settings', () => {
+    [true, false].forEach(async (checkedState) => {
+        test(`should render in rtl correctly when (checked: ${checkedState})`, async ({page, mount}) => {
+            await page.evaluate(() => {
+                document.documentElement.setAttribute('dir', 'rtl');
+            });
```

Comment:
I think this is a good direction to go in (no pun intended). Can you either:

a. extract this to a reusable function for other visual tests to utilize
b. create a ticket to do so?

If you plan to do this in a follow up PR then no worries 👍

Reply:
Good suggestion. So i think for now if we can leave it as it is that would be great. I've got a ticket which has been refined to look at all the missing RTL visual tests, I am now going to update that ticket with reference to this solution (or possibly after we have the reusage method). I think we also need a consider where the reusable function should live (most likely inside the pie-webc-testing package imo) so ill create a ticket for this today and get it refined.